resources/js/components/ThemeOverview.vue

```diff
@@ -0,0 +1,193 @@
+                        .get(`/api/topics?district=${currentDistrictName}`)
+                        .then(res => {
+                            if (res.data.data.length === 0) {
+                                this.info(currentDistrictName, this.districtInfoDescription);
+                            } else {
+                                res.data.data.forEach((topic) => {
+                                    if (topic.newTopic) {
+                                        this.themeListNewCount++;
+
+                                        if (this.themeListNew.length < 3) {
+                                            this.themeListNew.push(topic);
+                                        }
+                                    } else if (topic.finished) {
+                                        this.themeListFinishedCount++;
+
+                                        if (this.themeListFinished.length < 3) {
+                                            this.themeListFinished.push(topic);
+                                        }
+                                    } else {
+                                        this.themeListProgressCount++;
+
+                                        if (this.themeListProgress.length < 3) {
+                                            this.themeListProgress.push(topic);
+                                        }
+                                    }
+
+                                    // for filter block
+                                    if (topic.location.postalCode) {
+                                        if (!this.postcodeList.includes(topic.location.postalCode)) {
+                                            this.postcodeList.push(topic.location.postalCode);
+                                        }
+                                    }
+                                });
+                            }
+                        })
+                        .finally(() => {
+                            this.loading = false;
+
+                            if (this.firstLoading) {
+                                this.hideDefaultBlocks();
+                                this.firstLoading = false;
+                            }
+                        });
+                });
+
+                this.scrollTo(this.themeFirstBlock);
+            }
+        },
+        scrollTo(selector) {
+            const el = document.querySelector(selector);
+            if (el) {
+                el.scrollIntoView({ behavior: 'smooth'});
+            }
+        },
+        hideDefaultBlocks() {
+            const defaultBlocks = [
+                this.$refs.defaultThemeListNewBlock,
+                this.$refs.defaultThemeListProgressBlock,
+                this.$refs.defaultThemeListFinishedBlock,
+                this.$refs.defaultDistrictListBlock
+            ];
+
+            defaultBlocks.forEach(theme => {
+                if (theme) {
+                    theme.style.display = 'none';
```

Comment:
i think more efficient would be to use css class which apply display:none or v-show directive instead of creating reference for each block, loop through them and change their display prop